scikeras/utils/transformers.py

```diff
@@ -368,7 +368,7 @@ def inverse_transform(self, y: np.ndarray) -> np.ndarray:
             Keras Model predictions cast to the dtype and shape of the input
             targets.
         """
-        if self._y_dtype == np.float64 and y.dtype == np.float32:
+        if y.dtype == np.float32:
```

Comment:
```suggestion
        if y.dtype.name == "float32":
```

Reply:
Glad to put the change into #155, but this is just a garbage branch I created because tests were failing on Windows.

I can't test locally on windows (I have MacOS), so I created this branch to avoid polluting your notifications and the commit log in #155 while I test fixes for Windows by pushing to GH.